app/cdap/components/WrangleHome/index.tsx

```diff
@@ -23,17 +23,25 @@ import WrangleHomeTitle from 'components/WrangleHome/Components/WrangleHomeTitle
 import { GradientLine, HeaderImage } from 'components/WrangleHome/icons';
 import { useStyles } from 'components/WrangleHome/styles';
 import T from 'i18n-react';
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import { CARD_COUNT } from 'components/WrangleHome/constants';
 
 export default function() {
   const classes = useStyles();
-
+  const [showExplorations, setShowExplorations] = useState<boolean>(false);
   useEffect(() => {
     getWidgetData();
   }, []);
 
   return (
-    <Box className={classes.wrapper} data-testid="wrangler-home-new-parent">
+    <Box
+      className={
+        showExplorations ? classes.wrapper : `${classes.wrapper} ${classes.wrapperWithBottomSpace}`
```

Comment:
`${classes.wrapper}  ${ showExplorations && classes.wrapperWithBottomSpace}`
plz check if this kind of expression works